parastell/parastell.py

```diff
@@ -14,6 +14,11 @@
 from . import cubit_io
 from .utils import read_yaml_config, filter_kwargs, m2cm
 
+build_cubit_model_allowed_kwargs = ['skip_imprint', 'legacy_faceting']
+export_allowed_kwargs = ['faceting_tolerance', 'length_tolerance',
```

Comment:
Use more descriptive variable name.
```suggestion
export_dagmc_allowed_kwargs = ['faceting_tolerance', 'length_tolerance',
```

Reply:
i like it